### DISCUSSION TITLE
How to get generatede HTML CSS by grapesjs

### DISCUSSION THREAD

**Discussion by Ahsan763** [GrapesJS/grapesjs#6034] [Q&A]
![image](https://github.com/user-attachments/assets/0d422224-fd27-4cf9-9e94-8d8e8abbf970)
how i can get this full code in variables. i want to stor them in database

**Comment by mosh-tudor** [comment]
@Ahsan763 Check https://grapesjs.com/docs/modules/Storage.html#project-data

**Reply by glaprida** [reply]
Saving the project data is one thing, and getting the generated HTML, CSS and JS (already parsed, ready to print as static website) is another.

You can do it via `editor.getHtml()` ,  `editor.getCss()` and `editor.getJs()` . Those are the functions that the export plugin use in the background.

They each have some useful options: [More here](https://grapesjs.com/docs/api/editor.html#gethtml)
